resources/js/mosaic.tile.js

```diff
@@ -524,6 +524,14 @@ class Tile {
             if (onmousedown != undefined) {
                 btn.addEventListener("mousedown", onmousedown.bind(this));
             }
+            // Ensure keyboard activation (Enter / Space) triggers the same action
+            btn.addEventListener("keydown", function (ev) {
+                if (ev.key === "Enter" || ev.key === " ") {
+                    ev.preventDefault();
+                    // trigger the click handler attached above
+                    btn.click();
+                }
+            });
```

Comment:
In my tests this event never gets fired, because tile buttons are not accessible right now via keyboard.